test/pom.xml

```diff
@@ -86,6 +86,10 @@
       <groupId>org.apache.accumulo</groupId>
       <artifactId>accumulo-gc</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.accumulo</groupId>
+      <artifactId>accumulo-hadoop-mapreduce</artifactId>
+    </dependency>
```

Comment:
We shouldn't add this here. The mapreduce module can contain its own ITs, which have a test depenency on `accumulo-test`, for the testing framework stuffs.